backend/app/mailers/enrollment_mailer.rb

```diff
@@ -51,6 +51,36 @@ def notification_email_to_instructors
     )
   end
 
+  def notify_instructors_submitted_enrollment
+    @target_api_label = data_provider_config["label"]
+    @enrollment = Enrollment.find(params[:enrollment_id])
+    @demandeur_email = @enrollment.demandeurs.pluck(:email).first
+    @url = "#{ENV.fetch("FRONT_HOST")}/#{params[:target_api].tr("_", "-")}/#{params[:enrollment_id]}"
+
+    mail(
+      to: @enrollment.subscribers.pluck(:email),
+      from: "notifications@api.gouv.fr",
+      subject: "Nouvelle demande d’habilitation sur DataPass",
+      template_path: "enrollment_mailer/instructor",
+      template_name: "notify_submitted"
+    )
+  end
+
+  def notify_instructors_submitted_changes_requested
+    @target_api_label = data_provider_config["label"]
+    @enrollment = Enrollment.find(params[:enrollment_id])
+    @demandeur_email = @enrollment.demandeurs.pluck(:email).first
+    @url = "#{ENV.fetch("FRONT_HOST")}/#{params[:target_api].tr("_", "-")}/#{params[:enrollment_id]}"
+
+    mail(
+      to: @enrollment.subscribers.pluck(:email),
+      from: "notifications@api.gouv.fr",
+      subject: "Vous avez un retour sur votre demande de changement",
```

Comment:
I'd rather keep the imperative form for more consistence and conciseness. Plus, I'd keep the same wording everywere for this action "demande de changement" => "demande de modification".

```suggestion
      subject: "Retour sur votre demande de modification",
```